src/SpectatorView.Unity/Assets/SpectatorView/Scripts/StateSynchronization/BroadcasterSettings.cs

```diff
@@ -37,5 +37,17 @@ public bool ForcePerformanceReporting
         {
             get { return forcePerformanceReporting; }
         }
+
+        [SerializeField]
+        [Tooltip("Check to force loading all assets in asset caches on initialization. Note: this will improve the performance of identifying assets for sending asset changes between devices, but it will require using more memory.")]
+        private bool forceLoadAllAssetsDuringInitialization = false;
+
+        /// <summary>
+        /// Check to force loading all assets in asset caches on initialization. Note: this will improve the performance of identifying assets for sending asset changes between devices, but it will require using more memory.

```

Comment:
>Check to force [](start = 12, length = 14)

Nit: check to force -> forces

This is mirror the language of forcePerformanceReporting, and also because for getters there's nothing to check.